Feature/harubins/add multiline pattern config (#26460)
open-telemetry/opentelemetry-collector-contrib: pkg/stanza/docs/operators/file_input.md

The `multiline` configuration block must contain exactly one of `line_start_pattern` or `line_end_pattern`. These are regex patterns that
match either the beginning of a new log entry, or the end of a log entry.

> You can set the `omit_pattern_from_record` in case you want to omit the start/end pattern from each entry.

Reviewer: Nit, I think we don't typically address the reader directly. I'd prefer to describe the functionality like this.

```suggestion
The `omit_pattern` setting can be used to omit the start/end pattern from each entry.
```
